ui/admin/config/features.js

```diff
@@ -23,12 +23,10 @@ const baseEdition = {
   'byow-pki-hcp-cluster-id': false,
   'json-credentials': false,
   'static-credentials': false,
-  'target-worker-filters-v2-ingress': false,
-  'target-worker-filters-v2-hcp': false,
   'target-network-address': false,
-  'vault-worker-filter': false,
```

Comment:
looks like places that use the `vault-worker-filter` also need to be refactored ([example](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/components/credential-stores/credential-store/nav/index.hbs#L11))

Reply:
oh shoot, thank you! I missed that one 🙈